wren-ai-service/docker/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       WREN_AI_SERVICE_PORT: ${WREN_AI_SERVICE_PORT}
       OPENAI_API_KEY: ${OPENAI_API_KEY}
       OPENAI_API_BASE: ${OPENAI_API_BASE}
-      OPENAI_GENERATION_MODEL: ${OPENAI_GENERATION_MODEL}
```

Comment:
I think you need to make the same change to .env.example file as well

Reply:
yes, this is reflected in the .env.example file